dev-util/bugbite-cli/bugbite-cli-0.0.12-r1.ebuild

```diff
@@ -28,14 +29,15 @@ pkg_setup() {
 		export RUSTFLAGS="-C target-feature=+crt-static ${RUSTFLAGS}"
 		export static_stuff="--target ${rust_target}"
 	fi
+	rust_pkg_setup
 }
 
 src_configure() {
 		cargo_src_configure ${static_stuff}
 }
 
 src_test() {
-	edo cargo nextest run $(usev !debug '--release') --color always --features test --tests ${static_stuff}
+	${CARGO} nextest run $(usev !debug '--release') --color always --features test --tests ${static_stuff} || die
```

Comment:
Why not just edo ${CARGO}? You've also not then dropped the inherit for edo, but I don't think you should be changing that sort of style thing here.